.golangci.yaml

```
@@ -88,6 +88,11 @@ linters-settings:
     replace-allow-list:
       - "github.com/coder/websocket" # Currently uses our fork with a bug fix.
 
+  # Checks for potential security issues
+  gosec:
+    excludes:
+    - G115 # Potential integer overflow when converting between integer types
```

This is valid YAML, but to keep it consistent with the rest of the file I think there should be an extra indent here:
```suggestion
      - G115 # Potential integer overflow when converting between integer types
```